config/base/manager.yaml.in

```diff
@@ -30,17 +30,17 @@ spec:
           type: RuntimeDefault
       containers:
         - name: controller
-          image: quay.io/skupper/controller:v2-dev
+          image: ${IMAGE_REGISTRY}/${REPOSITORY_NAME}/controller@${CONTROLLER_SHA}
```

Comment:
I am not sure if it is a good idea to use only digests here.

The reason for my concern is that for non stable release versions,
like `v2-dev`, when the `v2-dev` tag reference is updated to a new digest,
the previous SHA that was referred by this tag will be removed by quay.io
if it does not have any other tag referring to it.

Maybe better to use tags here, and leave digests only for the related images?

Reply:
I was following the v1 operator as an example where it seems that the digests are being used
 
https://github.com/skupperproject/skupper-operator/blob/main/bundle/manifests/1.8.3/skupper-operator.v1.8.3.clusterserviceversion.yaml

I guess the question is if the bundle is updated on a release "boundary" shouldn't it be coupled to the digest as there could be other changes on v2-dev that affect the operator (e.g. a role change, etc.).

Agree that there are benefits to using the tags instead.

Reply:
My concern here is not with the bundle itself. From a bundle perspective it is fine to use just the SHA, but
the `base` folder is also used to produce the `skupper-cluster-scope.yaml`, which is built as part of the
`main` branch release. It has references to the `v2-dev` images.

Basically for main branch releases, where the tag is replaced with a new digest everytime something is merged
to main, we cannot just use digests as they will disappear from quay.io once the v2-dev tag is updated.

Reply:
Another idea could be to keep the /config directory with the kustomize files agnostic to the operator bundle: after all, that dir is used to generate the operator, the helm chart and the big deployment yamls on each release. How about modify what is needed for the operator with a script (like the helm chart generator script is doing)?